packages/manager/modules/vps/src/veeam/vps-veeam.html

```diff
@@ -7,11 +7,57 @@
     </div>
 
     <div data-ng-if="!$ctrl.veeam.loading">
+
+        <!--Backup Schedule section-->
+        <div class="mb-3" data-ng-if="$ctrl.isAutomateBackupAvailable">
```

Comment:
Yes, it's true it will empty.
The issue is that i can't make the div depends on logical conditions of sub elements.
```
if (FeatureFlipping)
   if (isValidVersion... && hasAutomated...)
        display section reschedule backup
   if (!isValidVersion)
        display message info
```
As you can see @JeremyDec we cant avoid the empty case.
Do you have an idea ?

Reply:
One solution would to ensure that whether you have `isValidVersion  && hasAutomated` || `!isValidVersion` condition on parent div element, but it looks ugly to do so....

Reply:
Yes that why i don't do it.